blazesym-go/cmd/cli/main.go

```diff
@@ -0,0 +1,145 @@
+package main
```

Comment:
I'd really not want to have a CLI like this floating around, as it will cause confusion with the Rust one. Can we make this an example of sorts instead? I.e., move it into the newly created `examples/` or so.

Reply:
I moved it to an example. This is how it looks like locally with `go doc -C go -http`:

<img width="1440" height="731" alt="image" src="https://github.com/user-attachments/assets/8e16aa9b-efae-4e1f-8117-b1df144a9735" />